client/src/components/NavBar.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Image from './Image';
+import StyledNavBar from './styled/StyledNavBar';
+import { Navbar, Nav, NavItem } from 'react-bootstrap';
+
+const NavBar = ({ children }) => {
+    return (
+        <StyledNavBar>
+            <Navbar.Header>
+                <Navbar.Brand>
+                     <a href="/">
```

Comment:
Can we use the <Link> component here?

Reply:
Is navbar both a custom component and a react-bootstrap component?

Reply:
Ah yeah I see the naming convention is confusing.. Navbar is a react-bootstrap component and NavBar is a custom component. I can rename NavBar, any suggestions on what to rename it?

Reply:
It's a bit confusing but if it works I'm not sure I have an issue with the naming, but if we are taking new names I vote CatNavBar